client/app/templates/components/contact-form.hbs

```diff
@@ -0,0 +1,36 @@
+{{#unless districtContactSubmitted}}
+  {{#unless districtContactErrored}}
+    <form {{action 'submitDistrictContact' on='submit'}}>
+        <div class="control-group">
+          <div class="control-label">
+            <label>Name</label>
+          </div>
+          <div class="controls">
+            {{input value=name name='Name' type="text"}}
+          </div>
+        </div>
+        <div class="control-group">
+            <div class="control-label">
+              <label>Email</label>
+            </div>
+            <div class="controls">
+              {{input value=districtContactEmail name='Email' required=true type="email"}}
+            </div>
+        </div>
+        <div class="control-group">
+          <div class="control-label">
+            <label>Message</label>
+          </div>
+          <div class="controls">
+            {{textarea value=message type="text" name='Message' rows="6" placeholder="We want schoolbot!"}}
```

Comment:
The rest of the app uses the capitalization "SchoolBot" in prose, so we should be consistent here.